bridge-token/src/lib.rs

```diff
@@ -4,19 +4,14 @@ use near_contract_standards::fungible_token::metadata::{
 };
 use near_contract_standards::fungible_token::FungibleToken;
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
-use near_sdk::json_types::{Base64VecU8, ValidAccountId, U128};
+use near_sdk::json_types::{Base64VecU8, U128};
 use near_sdk::{
     assert_one_yocto, env, ext_contract, near_bindgen, AccountId, Balance, Gas, PanicOnDefault,
     Promise, PromiseOrValue, StorageUsage,
 };
-use std::convert::TryInto;
-
-near_sdk::setup_alloc!();
-
-const NO_DEPOSIT: Balance = 0;
 
 /// Gas to call finish withdraw method on factory.
-const FINISH_WITHDRAW_GAS: Gas = 50_000_000_000_000;
+const FINISH_WITHDRAW_GAS: Gas = Gas(50_000_000_000_000);
```

Comment:
Please use the following pattern for readability
```suggestion
const FINISH_WITHDRAW_GAS: Gas = Gas(Gas::ONE_TERA.0 * 50);
```

Reply:
Fixed 8e2b316371f19907869b1bd8506f976beb82ce31